src/praisonai-train/praisonai_train/data/export_trials.py

```
@@ -0,0 +1,308 @@
+
+
+def _to_alpaca(turns: list[dict[str, str]]) -> dict[str, str]:
+    """Flatten ShareGPT turns to instruction/input/output (Alpaca)."""
+    system = "".join(t["content"] for t in turns if t["role"] == "system")
+    users = [t["content"] for t in turns if t["role"] == "user"]
+    output = next(
+        (t["content"] for t in reversed(turns) if t["role"] == "assistant"), ""
+    )
+    return {
+        "instruction": system or (users[0] if users else ""),
+        "input": "\n".join(users[1:]) if system else "\n".join(users[1:]),
```

Ruff (0.15.21)

[warning] Useless `if`-`else` condition

(RUF034)